.rubocop.yml

```diff
@@ -18,3 +16,12 @@ Style/MutableConstant:
   Enabled: false
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+
+MethodLength:
+  Max: 12
```

Comment:
In my opinion it is not enough length.
Example name: `unread_message_count` - 20 symbols.
 What about 25 ?